pkgs/by-name/ce/cen64/package.nix

```
@@ -1,53 +1,56 @@
 {
   lib,
-  cmake,
+  stdenv,
   fetchFromGitHub,
+  cmake,
   libGL,
   libiconv,
   libX11,
   openal,
-  stdenv,
+  nix-update-script,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "cen64";
-  version = "0-unstable-2023-05-29";
+  version = "0.3-unstable-2025-10-24";
 
   src = fetchFromGitHub {
     owner = "n64dev";
     repo = "cen64";
-    rev = "1c1118462bd9d9b8ceb4c556a647718072477aab";
-    sha256 = "sha256-vFk29KESATcEY0eRNbS+mHLD9T1phJiG1fqjOlI19/w=";
+    rev = "e0641c8452a3ae8edcd2bf4e46794bb4eaafc076";
+    hash = "sha256-PpaD3hgksPD729LyFm7+ID8i+x3yZ0f+S11eSQyoB64=";
   };
 
-  patches = [
-    # fix build with gcc14:
-    # https://github.com/n64dev/cen64/pull/191/commits/f13bdf94c00a9da3b152ed9fe20001e240215b96
-    ./cast-mi_regs-callbacks.patch
-    # https://github.com/n64dev/cen64/pull/237
-    ./fix-thread-arg-type-for-pthread_setname_np.patch
-  ];
+  # fix build with gcc14:
+  # https://github.com/n64dev/cen64/pull/191/commits/f13bdf94c00a9da3b152ed9fe20001e240215b96
+  patches = [ ./cast-mi_regs-callbacks.patch ];
 
+  strictDeps = true;
   nativeBuildInputs = [ cmake ];
   buildInputs = [
     libGL
     libiconv
-    openal
     libX11
+    openal
   ];
 
   installPhase = ''
     runHook preInstall
-    install -D {,$out/bin/}${pname}
+
+    install -D ${finalAttrs.meta.mainProgram} \
+      --target-directory=$out/bin
+
     runHook postInstall
   '';
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { };
```

```suggestion
  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };
```

If you want nix-update-script to use the latest commit as the update instead of failing because it can't find a tag.